### DISCUSSION TITLE
useSlots is not defined

### DISCUSSION THREAD

**Discussion by skoenfaelt** [vuejs/test-utils#2223] [Q&A] [answered]
According to this Issue https://github.com/vuejs/vue-test-utils/issues/2033
Is there any solution for `"@vue/test-utils": "^2.4.1",`?

I can not test components with useSlots()...

**Comment by cexbrayat** [comment]
You're linking an issue for VTU v1 which is for Vue 2.
This repository is for VTU v2 which is for Vue 3. I don't think we have an issue with `useSlots`.
If you have one, can you provide us a small repro online using https://stackblitz.com/github/vuejs/create-vue-templates/tree/main/typescript-vitest?file=src%2Fcomponents%2F__tests__%2FHelloWorld.spec.ts ? 

It only takes a few minutes, and we'll be able to take a look

**Reply by skoenfaelt** [reply]
I know that the issue is for VTU 1.
In my context I test a Nuxt3 App with vitest, vue test utils 2. I will look how to give you a working reproduction.

**Comment by skoenfaelt** [comment, marked as answer]
Excuse me.

The Issue is in Nuxt Auto Imports of Vue. Ref, UseSlots... needs to be imported into the component. The Auto Import Feature does not work in Test context

**Reply by cexbrayat** [reply]
Yes, you need to import it, or set up your tooling to add the import for you. Maybe https://github.com/danielroe/nuxt-vitest may help you
